packages/eslint-plugin/tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "allowJs": true,
     "checkJs": true,
     "module": "commonjs",
-    "moduleResolution": "node",
+    "moduleResolution": "nodenext",
```

Comment:
typedefs imports no longer detected with `moduleResolution: node` from `@typescript-eslint/utils`

[typescript-eslint#7185](https://github.com/typescript-eslint/typescript-eslint/issues/7185)
[typescript-eslint#7284](https://github.com/typescript-eslint/typescript-eslint/issues/7284)

Reply:
hmm interesting... I understand the push though, luckily this is only for 1 package ATM. having this as a react ui package requirement we would have a problem. AFAIR there is no way hot to migrate to this gradually